kernel/packages/shared/dao/index.ts

```diff
@@ -451,11 +452,12 @@ export function observeRealmChange(
   onRealmChange: (previousRealm: Realm | undefined, currentRealm: Realm) => any
 ) {
   let currentRealm: Realm | undefined = getRealm(store.getState())
-  store.subscribe(() => {
+  store.subscribe(async () => {
     const previousRealm = currentRealm
     currentRealm = getRealm(store.getState())
     if (currentRealm && !deepEqual(previousRealm, currentRealm)) {
       onRealmChange(previousRealm, currentRealm)
+      await getJWT(currentRealm)
```

Comment:
Would it make sense to add this as a subscriptor instead of directly adding it in this method?